src/app/components/termofservice/index.tsx

```diff
@@ -0,0 +1,320 @@
+Chain Transaction”. Recovery of Cross-Chain Transactions is a risky and time-consuming process. The level of complexity and risk depends both on the type of digital asset and the address used for the Transaction. Not all Cross-Chain Transactions are recoverable, and digital assets sent to the wrong address are at risk of recovery by bad actors not associated with Vultisig. 
+</p>
+<p>
+Vultisig IS NOT RESPONSIBLE OR LIABLE FOR ANY CROSS-CHAIN TRANSACTIONS INITIATED BY YOU OR ANY OTHER USER. If you request assistance in recovering a Cross Chain Transaction, Vultisig may decide in its sole discretion whether to assist or attempt to recover such Cross-Chain Transaction. There is no time limit for any such discretionary recovery attempts and Vultisig will not be liable for a failed recovery of any Cross-Chain Transaction. If any such recovery is attempted, Vultisig may assess you with a fee, communicated to you in advance, that reflects the level of effort and developer time consumed on the amount or value of the digital asset. You acknowledge and consent to the assessment of this fee when making any such recovery request. 
+</p>
+<p>If you would like to ask Vultisig to attempt to recover your digital assets from a Cross-Chain Transaction that you initiated, please reach out to our support team with the details of the Transaction by emailing our customer support info@vultisig.com. Vultisig will evaluate your request and determine in its sole discretion whether to initiate recovery procedures. You acknowledge that even if Vultisig decides to attempt a Cross-Chain Transaction recovery on your behalf, we make no warranty and/or guarantee that any such attempted recovery will ultimately be successful.
+</p>
+                        </li>
+                    </ol>
+                    </li>
+                    <li>
+                    VULT Tokens 
+                   <ol type="a">
+                    <li>
+                    VULT Token Basics. The VULT token is a digital asset that Vultisig may issue in the future, with plans for an Initial Liquidity Offering (ILO). It is intended to be a standard ERC-20 token on the Ethereum blockchain, accessible through most wallets that support Ethereum. Vultisig plans to use VULT tokens to provide loyalty benefits in connection with a User’s use of the Service, such as enhanced features. Vultisig may change, add, or remove any benefits associated with VULT tokens at any time, in its sole discretion. Each User’s VULT token balance will be displayed in Vultisig, and Vultisig may place restrictions on VULT tokens’ interaction with the Service, including minimum use or withdrawal limits.
```

Comment:
```suggestion
                    VULT Token Basics. The VULT token is a digital asset that Vultisig may issue in the future. It is intended to be a standard ERC-20 token on the Ethereum blockchain, accessible through most wallets that support Ethereum. Vultisig plans to use VULT tokens to provide loyalty benefits in connection with a User’s use of the Service, such as enhanced features. Vultisig may change, add, or remove any benefits associated with VULT tokens at any time, in its sole discretion. Each User’s VULT token balance will be displayed in Vultisig.
```

Please update like this